src/lib/receiver.js

```diff
@@ -118,10 +121,12 @@ function createReceiver (opts) {
     const paymentRequest = {
       address: packet.account,
       amount: packet.amount,
-      expires_at: packet.data.expires_at,
-      data: packet.data.data
+      expires_at: packet.data.expires_at
```

Comment:
in that case you would end up with a `{data: false}` which is still unnecessary.